### PR TITLE
Condense River Drake tail lash effects

### PR DESCRIPTION
**Pull request by CrackJackFlood**
Update river drake and other adjacent creatures who had multiple redundant effects

Effect: Tail Lash(Attack Roll)
Effect: Tail Lash(Skill Check)

are now

Effect: Tail Lash

### CODE REVIEW

File: packs/bestiary-effects/effect-tail-lash.json [hunk 1]
```diff
@@ -26,6 +26,12 @@
                 "selector": "attack",
                 "type": "circumstance",
                 "value": -2
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "skill-check",
+                "type": "circumstance",
+                "value": -2
```
Comment (pedrogrullada) [inline packs/bestiary-effects/effect-tail-lash.json:34]: You're gonna need a Choice Set in this effect if you want to condense them. The penalty doesn't apply to both Strikes and skill checks. It applies to the type of roll that triggered it. Consider adding a `"removeAfterRoll": "if-enabled"` so the effect is removed after the triggering roll is done.

**Comment by pedrogrullada** [comment]
If you're deleting an effect you need to redirect it, in addition to removing its references in other compendium items and actors. See #16592 for an example on how to do these uuid redirects if you don't know.

File: packs/bestiary-effects/effect-tail-lash.json (hunk 1 above)
Comment (TikaelSol) [reply packs/bestiary-effects/effect-tail-lash.json]: It only applies to the triggering check anyways

Comment (CrackJackFlood) [reply packs/bestiary-effects/effect-tail-lash.json]: > It only applies to the triggering check anyways

@pedrogrullada is right its supposed to only effect the triggering roll, I did the suggested changes.

File: packs/bestiary-effects/effect-tail-lash.json [hunk 2]
```diff
@@ -23,7 +23,11 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "selector": "attack",
+                "removeAfterRoll": "if-enabled",
```
Comment (TikaelSol) [inline packs/bestiary-effects/effect-tail-lash.json:26]:
```suggestion
                "removeAfterRoll": true,
```

File: build/uuid-redirects.json [hunk 3]
```diff
@@ -552,5 +552,6 @@
     "Compendium.pf2e.spell-effects.Item.yke7fAUDxUNzouQc": "Compendium.pf2e.spell-effects.Item.Spell Effect: Elemental Gift",
     "Compendium.pf2e.spell-effects.Item.yq6iu0Qxg3YEbb6s": "Compendium.pf2e.spell-effects.Item.Spell Effect: Elemental Gift",
     "Compendium.pf2e.spell-effects.Item.zdpTPf157rXl3tEJ": "Compendium.pf2e.spell-effects.Item.Spell Effect: Clawsong",
-    "Compendium.pf2e.the-slithering-bestiary.Actor.idj4GBmsLUNUbk9r": "Compendium.pf2e.pathfinder-bestiary.Actor.Alchemical Golem"
-}
+    "Compendium.pf2e.the-slithering-bestiary.Actor.idj4GBmsLUNUbk9r": "Compendium.pf2e.pathfinder-bestiary.Actor.Alchemical Golem",
+    "Compendium.pf2e.spell-effects.Item.4HJJdogS751Z04lD": "Compendium.pf2e.spell-effects.Item.Effect: Tail Lash"
+}
```
Comment (TikaelSol) [inline build/uuid-redirects.json:557]:
```suggestion
    "Compendium.pf2e.bestiary-effects.Item.4HJJdogS751Z04lD": "Compendium.pf2e.spell-effects.Item.Effect: Tail Lash"
}

```
I gave you the wrong path, wasn't paying attention there. This should be inserted in alphabetical order, but I can just do a sort on the file later. I'm sure it needs it anyways.

File: packs/kingmaker-bestiary/ilthuliak.json [hunk 4]
```diff
@@ -3179,4 +3179,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/kingmaker-bestiary/ilthuliak.json:3182]:
```suggestion
}

```

File: packs/pathfinder-bestiary/black-dragon-adult-spellcaster.json [hunk 5]
```diff
@@ -2080,4 +2080,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pathfinder-bestiary/black-dragon-adult-spellcaster.json:2083]:
```suggestion
}

```

File: packs/pathfinder-bestiary/black-dragon-adult.json [hunk 6]
```diff
@@ -727,4 +727,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pathfinder-bestiary/black-dragon-adult.json:730]:
```suggestion
}

```

File: packs/pathfinder-bestiary/black-dragon-ancient-spellcaster.json [hunk 7]
```diff
@@ -2587,4 +2587,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pathfinder-bestiary/black-dragon-ancient-spellcaster.json:2590]:
```suggestion
}

```

File: packs/pathfinder-bestiary/black-dragon-ancient.json [hunk 8]
```diff
@@ -735,4 +735,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pathfinder-bestiary/black-dragon-ancient.json:738]:
```suggestion
}

```

File: packs/pathfinder-bestiary/black-dragon-young-spellcaster.json [hunk 9]
```diff
@@ -1423,4 +1423,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pathfinder-bestiary/black-dragon-young-spellcaster.json:1426]:
```suggestion
}

```

File: packs/pathfinder-bestiary/black-dragon-young.json [hunk 10]
```diff
@@ -531,4 +531,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pathfinder-bestiary/black-dragon-young.json:534]:
```suggestion
}

```

File: packs/pathfinder-bestiary/river-drake.json [hunk 11]
```diff
@@ -360,4 +360,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pathfinder-bestiary/river-drake.json:363]:
```suggestion
}

```

File: packs/pathfinder-monster-core/river-drake.json [hunk 12]
```diff
@@ -361,4 +361,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pathfinder-monster-core/river-drake.json:364]:
```suggestion
}

```

File: packs/pfs-season-6-bestiary/demonic-ocean-drake.json [hunk 13]
```diff
@@ -613,4 +613,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pfs-season-6-bestiary/demonic-ocean-drake.json:616]:
```suggestion
}

```

File: packs/pfs-season-6-bestiary/demonic-river-drake.json [hunk 14]
```diff
@@ -432,4 +432,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/pfs-season-6-bestiary/demonic-river-drake.json:435]:
```suggestion
}

```

File: packs/quest-for-the-frozen-flame-bestiary/venexuss-chosen.json [hunk 15]
```diff
@@ -474,4 +474,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/quest-for-the-frozen-flame-bestiary/venexuss-chosen.json:477]:
```suggestion
}

```

File: packs/seven-dooms-for-sandpoint-bestiary/scarhorn.json [hunk 16]
```diff
@@ -2468,4 +2468,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (TikaelSol) [inline packs/seven-dooms-for-sandpoint-bestiary/scarhorn.json:2471]:
```suggestion
}

```

File: build/uuid-redirects.json [hunk 17]
```diff
@@ -552,5 +552,6 @@
     "Compendium.pf2e.spell-effects.Item.yke7fAUDxUNzouQc": "Compendium.pf2e.spell-effects.Item.Spell Effect: Elemental Gift",
     "Compendium.pf2e.spell-effects.Item.yq6iu0Qxg3YEbb6s": "Compendium.pf2e.spell-effects.Item.Spell Effect: Elemental Gift",
     "Compendium.pf2e.spell-effects.Item.zdpTPf157rXl3tEJ": "Compendium.pf2e.spell-effects.Item.Spell Effect: Clawsong",
-    "Compendium.pf2e.the-slithering-bestiary.Actor.idj4GBmsLUNUbk9r": "Compendium.pf2e.pathfinder-bestiary.Actor.Alchemical Golem"
+    "Compendium.pf2e.the-slithering-bestiary.Actor.idj4GBmsLUNUbk9r": "Compendium.pf2e.pathfinder-bestiary.Actor.Alchemical Golem",
+    "Compendium.pf2e.bestiary-effects.Item.4HJJdogS751Z04lD": "Compendium.pf2e.spell-effects.Item.Effect: Tail Lash"
```
Comment (TikaelSol) [inline build/uuid-redirects.json:556]:
```suggestion
    "Compendium.pf2e.bestiary-effects.Item.4HJJdogS751Z04lD": "Compendium.pf2e.bestiary-effects.Item.Effect: Tail Lash"
```

**Pull request merged**